sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+CustomUser = get_user_model()
+
+user_type_options = {"admin": "admin", "manager": "manager", "employee": "employee"}
+
+password_for_every_user = "userpasswd"
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-a", "--admin", type=int, help="Indicates the maximum number of admins to be in the database"
+        )
+        parser.add_argument(
+            "-e", "--employee", type=int, help="Indicates the maximum number of employees to be in the database"
+        )
+        parser.add_argument(
+            "-m", "--manager", type=int, help="Indicates the maximum number of managers to be in the database"
+        )
```

Comment:
I can see that the names of these arguments are the same as values under **user_type_options**. Therefore, why not linking them here and keep it all consistent? I believe this will also allow you to safely use those values further down, under the **handle** with the **options** argument, if I follow the code correctly?